### DISCUSSION TITLE
Model distillation?

### DISCUSSION THREAD

**Discussion by noe** [openai/whisper#752] [Q&A]
Is there available any source code or Colab notebook to distill the large models into a smaller one?

Thank you very much.

Update: my goal would be to use some of the large models finetuned for other languages and create smaller versions of them, first via distillation, then with quantization.

**Comment by glangford** [comment]
See related discussions https://github.com/openai/whisper/discussions/11 https://github.com/openai/whisper/discussions/454

**Reply by noe** [reply]
Thanks! These are about quantization, not distillation, though.

**Comment by rraihansaputra** [comment]
Hi @noe, I'm also interested in this for the same goal. Have you found resources related to this?
